fabric-recipe-api-v1/build.gradle

```diff
@@ -7,3 +7,7 @@ loom {
 moduleDependencies(project, [
 	"fabric-networking-api-v1",
 ])
+
+testDependencies(project, [
+		':fabric-lifecycle-events-v1',
+])
```

Comment:
Nit: Files should end with a newline.

Reply:
Will add the newline and also fix the indentation and quotes